notary-server/openapi.yaml

```diff
@@ -175,8 +175,11 @@ components:
           enum:
             - "Tcp"
             - "Websocket"
-        maxTranscriptSize:  
-          description: Maximum transcript size in bytes
+        maxSentData:  
+          description: Maximum data that can be sent by the prover in bits
```

Comment:
shouldnt this be bytes, not bits.

btw, where in the code does the server handle `maxSentData`? Grepping this crate didnt find where it is used.

Reply:
https://github.com/tlsnotary/tlsn/blob/dev/notary-server/src/domain/notary.rs#L24-L26 — just different casing type used, we convert snake_case used internally to camelCase in payload for clients (which I supposed is more standard)